docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -34,6 +34,28 @@ The json payload for create and update operations cannot contain keys beginning
 
 @@@
 
+## Nexus metadata
```

Comment:
I feel like the common metadata (so some related to resources can remain here) can live in their own page because they are also returned by `global` entities such as realms/orgs/...

Reply:
I extracted the global fields into their own page. I also added info on Files metadata since they have the most custom fields. I'm not sure if it's worth adding this for all entities (some of them like Projects will have `_uuid` and `_constrainedBy`)